joomla/joomla-cms
# [admin menus] script.php incorrectly deletes com_menus searchtools layouts

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/14159

### Summary of Changes

Preventing script.php from deleting com_menus searchtools layout

### Testing Instructions
Install staging. create an admin menu.
Go to Extensions->Manage->Database and click Fix
See #14159 for the results: the client filter does not show.

Patch and reinstall staging. 
Create admin menu and fix database
The problem should be solved.
we will normally get:
![adminmenu2](https://cloud.githubusercontent.com/assets/869724/23156350/d3c2f220-f816-11e6-86c9-c3e52e75bc2b.gif)
